indico/modules/events/timetable/forms.py

```diff
@@ -209,3 +209,7 @@ def data_for_format(self):
         for fieldname in fields:
             data.update(getattr(self, fieldname).data)
         return data
+
+
+class ImportContributionsForm(IndicoForm):
+    source_file = FileField(_("Source File"), accepted_file_types='.csv')
```

Comment:
in some places we specify a mime type for `accepted_file_types`, in others a file extension... not sure if there's any advantage of using one or the other.